qiskit/transpiler/target.py

```diff
@@ -1034,14 +1034,14 @@ def build_coupling_map(self, two_q_gate=None, filter_idle_qubits=False):
 
         Raises:
             ValueError: If a non-two qubit gate is passed in for ``two_q_gate``.
-            IndexError: If an Instruction not in the Target is passed in for
+            IndexError: If an Instruction not in the ``Target`` is passed in for
                 ``two_q_gate``.
         """
         if self.qargs is None:
             return None
         if None not in self.qargs and any(len(x) > 2 for x in self.qargs):
             logger.warning(
-                "This Target object contains multiqubit gates that "
+                "This Target object contains multi-qubit gates that "
```

Comment:
"multiqubit" is fine (especially in US English), it's just a constructed word.

Reply:
Also a small note here, should this change stay, the log assert in the target unit test would have to be modified to add the hyphen (this is why CI currently fails): https://github.com/Qiskit/qiskit/blob/main/test/python/transpiler/test_target.py#L558